### PR TITLE
Basic fixes to get finetune.sh working for Rag

### PR DESCRIPTION
**Pull request by richardliaw**
Fixes #{issue number}

### CODE REVIEW

File: examples/rag/finetune.py [hunk 1]
```diff
@@ -66,6 +70,25 @@ def __init__(self, *args, **kwargs):
         super(AttrDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
 
+# https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/backends/test_accelerator_connector.py
+
+class Accel(DDPAccelerator):
```
Comment (amogkam) [inline examples/rag/finetune.py:75]: Why is this necessary?

File: examples/rag/finetune.py [hunk 2]
```diff
@@ -328,12 +344,12 @@ def train_dataloader(self) -> DataLoader:
             // self.hparams.accumulate_grad_batches
             * float(self.hparams.max_epochs)
         )
-        scheduler = get_linear_schedule_with_warmup(
-            self.opt, num_warmup_steps=self.hparams.warmup_steps, num_training_steps=t_total
-        )
-        if max(scheduler.get_last_lr()) > 0:
-            warnings.warn("All learning rates are 0")
-        self.lr_scheduler = scheduler
+        # scheduler = get_linear_schedule_with_warmup(
```
Comment (amogkam) [inline examples/rag/finetune.py:347]: Oh right we can just comment out the scheduler if we don't care about accuracy. But we should still raise an issue for this right? This is a bug in their master branch.

File: examples/rag/finetune.py (hunk 1 above)
Comment (amogkam) [reply examples/rag/finetune.py:75]: It also doesn't seem to be used anywhere

Comment (richardliaw) [reply examples/rag/finetune.py:75]: Because of PTL DDP deprecation/migration to a different codepath in 1.0

File: examples/rag/finetune.py (hunk 2 above)
Comment (richardliaw) [reply examples/rag/finetune.py:347]: Yeah, that is a good idea (though wasn't a high priority for me yesterday)

Comment (richardliaw) [reply examples/rag/finetune.py:347]: Also, i think the scheduler is actually already set in `configure_optimizer` in lightningbase
